tests/functional/splinter_test.c

```diff
@@ -2608,6 +2620,13 @@ splinter_test(int argc, char *argv[])
       config_argc -= 1;
       config_argv += 1;
    }
+   if (config_argc > 0
+       && strncmp(config_argv[0], "--use-shmem", sizeof("--use-shmem")) == 0)
```

Comment:
Doesn't the generic config parsing handle this?

Reply:
@rtjohnso - Well, ... `config_parse()` does parse this `--use-shmem` arg, but that function is not being used in this test to parse cmdline args. This test has very painful parsing of fixed-positions of arguments. 

And somewhere along the line it's calling this even-further specialized `test_config_parse()` parsing function.

So it got difficult to use `config_parse()` to recognize use of `--use-shmem`.